hugo/content/news/2010-08-17-for-the-record.md

```diff
@@ -20,18 +20,18 @@ Let&#8217;s start with the basic rules we established for the recorder:
 * For privacy reasons we will not allow recording on pre 1.2.3 servers. (They are not able to properly warn their users)
 * There will be no ACLs to disallow it on newer servers. (We feel that if we restrict the feature that way the Open Source nature of our project will lead to forks with the warnings completely removed which is an even worse situation for the users)
 
-Once you are on a server that fulfils these prerequisites you can open the Recorder via our Icon toolbar or the menu (Self->Recorder) and &#8220;Start&#8221; the recording. You can record for as long as your disk space lasts or until you grow tired of it. The data is encoded and written away on the fly. Supported audio formats at this time are:
+Once you are on a server that fulfils these prerequisites you can open the Recorder via our Icon toolbar or the menu (Self->Recorder) and &#8220;Start&#8221; the recording. You can record for as long as your disk space lasts or until you grow tired of it. The data is encoded and written away on the fly. Supported audio formats at this time are:
```

Comment:
What is this kind of change about? Seems unrelated

Reply:
I have no clue, but it seems that spaces were added or not?

Reply:
Not really. It rather seems like spaces have been replaced. In any case these unrelated changes should be removed from this PR.

Reply:
Well, I will see if I have time and interest to check all the files for that.
I use a completely normal program (ghostwriter), so I don't know what the reasons for this are exactly.